### ISSUE TITLE
History search bar hides but doesn't reset

### ISSUE THREAD

**Issue by ghost** [Adamantcheese/Kuroba#1370]
Similar to #1274, the search bar acts like it hasn't been used but the history sidebar keeps the same results it was showing before clicking on a thread

Tested on v4.13.6-34a61da1 on a Samsung Note 10 Plus

**Comment by Adamantcheese** [comment]
You're going to have to explain better or provide some reproduction steps because I can't reproduce whatever I think you're talking about. I would think that keeping the search around in the history drawer is a better thing to do than to completely clear it out when you enter a thread from history. You might not find what you want in that thread or want to browse other stuff more quickly by keeping the search results after entering a thread.

**Comment by ghost** [comment]
> You're going to have to explain better or provide some reproduction steps because I can't reproduce whatever I think you're talking about. I would think that keeping the search around in the history drawer is a better thing to do than to completely clear it out when you enter a thread from history. You might not find what you want in that thread or want to browse other stuff more quickly by keeping the search results after entering a thread.

I agree that it's better to keep the bar open, but I'm saying that it removes the searched text but keeps the results. So you have to press the search button again to modify your search

It doesn't happen every time but I guess I will show a screenshot next time it happens to me
